dotnetv3/AutoScaling/Actions/HelloAutoScaling.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[AutoScaling.dotnetv3.AutoScalingActions.HelloAutoScaling]
+namespace AutoScalingActions;
+
+public class HelloAutoScaling
+{
+    /// <summary>
+    /// Hello Amazon EC2 Auto Scaling. List EC2 Auto Scaling groups.
+    /// </summary>
+    /// <param name="args"></param>
+    /// <returns>Async Task.</returns>
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon service.
+        using var host = Host.CreateDefaultBuilder(args)
+            .ConfigureServices((_, services) =>
+                services.AddAWSService<IAmazonAutoScaling>()
+            )
+            .Build();
+
+        var client = host.Services.GetRequiredService<IAmazonAutoScaling>();
```

Comment:
This was probably written before we stopped using DI for the Hello Service, but you can use the simpler setup to initialize the client directly, like in the Hello IAM example.

Reply:
Updated it to not using DI. Yes, the code was written before the change. Thanks for pointing it out.